### ISSUE TITLE
New install won’t allow log in

### ISSUE THREAD

**Issue by cdedert** [YunoHost-Apps/friendica_ynh#146]
### Describe the bug
I installed Friendica and it states that I should be able to access the login page, then use my Yunohost credentials to continue logging in. I do get the login page, but nothing seems to happen when I try to log in. I don’t get a rejected password or rejected username, the page just does nothing. I did notice that in the Services, the Friendica-daemon will not run. Seems to fail and continuously restart. Here are the logs. https://paste.yunohost.org/ujewegepid

### Context
- Hardware: Dell Poweredge R620
- YunoHost version: 11.2.30
- I have access to my server: Through SSH | through the webadmin | direct access via keyboard / screen | 
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: no
- Using, or trying to install package version/branch: Friendica 2024.08~ynh1

### Steps to reproduce
Install Friendica using the web admin. Once installed, use Yunohost credentials to gain access to and use Friendica. Login is not successful.

I uninstalled Friendica, and then re-install using the CLI, but same results as above. Noticed that the Friendica-daemon service would not run properly.

### Expected behavior
Use webadmin to install Friendica. Once installed, gain access as Admin to the Friendica instance. 

### Logs
https://paste.yunohost.org/ujewegepid

**Comment by cdedert** [comment]
Help still needed. Thanks.

**Comment by LPS-github** [comment]
Just posting here to let you know you're not alone, I experienced the same issue after a fresh install.  

I'm being told the install was successful, it then instructs me to visit the domain and login using the yunohost user I selected during install and my regular password.  Unfortunately no luck:(


![Friendica _ Applications _ YunoHost Admin — Mozilla Firefox](https://github.com/user-attachments/assets/488f8cf9-ce39-42fc-9591-22cd5d66013e)

**Comment by cdedert** [comment]
I never did get it to work. I ended up installing Hubzilla instead and am happy with that.

**Comment by LPS-github** [comment]
Thanks for the update, it's really too bad because Friendica seems like a simplified version of Hubzilla and add to that the fact that it can connect to diaspora as well is a huge bonus.

**Comment by cdedert** [comment]
Hubzilla is more complex. But also, it has a Diaspora plugin that allows that connections to be made.

**Comment by abeorch** [comment]
I was just looking at https://forum.yunohost.org/t/friendica-logging-in-as-admin-after-new-installation/28034/3 which is a little old but suggetst that you have to **create** a friendica user - with an email that matches the yunohost admin email account - I'm not sure if this is still relevant but it appears for me at least that I can't create users in Friendica as the /register page is not available.

**Comment by abeorch** [comment]
I think I might have figured out what has happened - After creating the Friendica Instance I noticed that I couldn't login but that also there was a Friendica daemon not running  - this led me to find https://forum.yunohost.org/t/again-friendica-daemon-not-starting/29889 which after following I was able to get Friendica to run and I was able to login (with one issue - If I used the same password as I used for yunohost - I got blocked with a message saying that the password had been found in a list of compromised passwords (which I think is strange since I only created it a few minutes ago )

**Comment by isAAAc** [comment]
yes ! thx @abeorch , perhaps we should explain this in the Readme.md as "post-install workaround" waiting a real fix ?

**Comment by abeorch** [comment]
> yes ! thx @abeorch , perhaps we should explain this in the Readme.md as "post-install workaround" waiting a real fix ?

That seems a reasonable idea to me (but I'm no expert)

**Comment by trendless** [comment]
Could these three lines during the install hint at what's causing the database to need to `dbstructure` command to be manually run after install before login is possible?

`Warning: Invalid file '/usr/sbin/mysqld' for capability operation`
`Warning: Setcap failed on /usr/sbin/mysqld, required with --memlock if insufficent RLIMIT_MEMLOCK`
`Warning: rsync.service is a disabled or a static unit, not starting it.`

**Comment by Plumeros** [comment]
Same here.
